src/kaboom.ts

```diff
@@ -3159,6 +3233,14 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 		}
 	}
 
+	winEvents.gamepadconnected = (e) => {
+		game.ev.trigger("gamepadConnect", e.gamepad)
+	}
+
+	winEvents.gamepaddisconnected = (e) => {
+		game.ev.trigger("gamepadDisconnect", e.gamepad)
```

Comment:
I usually put all input related handlers in `game.ev.onOnce("input)`, to handle all input events at once in a frame instead of handling inputs in random phases in a frame

Reply:
Will change :)